src/datagen/generated/minecolonies/data/minecolonies/tags/items/reduceable_ingredient.json

```diff
@@ -11,8 +11,8 @@
     "#forge:cobblestone",
     "#forge:gravel",
     "#forge:sand",
-    "#forge:dusts/redstone",
-    "#forge:gems/diamond",
+    "#forge:dusts",
+    "#forge:gems",
```

Comment:
Here's one change -- it seemed reasonable to allow other dusts and gems to be reduced in improved recipes.